matrix_sdk_base/src/store/memory_store.rs

```diff
@@ -502,3 +502,111 @@ impl StateStore for MemoryStore {
         self.get_event_room_receipt_events(room_id, receipt_type, event_id).await
     }
 }
+
+#[cfg(test)]
+#[cfg(not(feature = "sled_state_store"))]
+mod test {
+    use matrix_sdk_common::{
+        identifiers::{event_id, room_id, user_id},
+        receipt::ReceiptType,
+    };
+    use matrix_sdk_test::async_test;
+    use serde_json::json;
+
+    use super::{MemoryStore, StateChanges};
+
+    #[async_test]
+    async fn test_receipts_saving() {
+        let store = MemoryStore::new();
+
+        let room_id = room_id!("!test:localhost");
+        let user_id = user_id!("@rikj:jki.re");
```

Comment:
Let's just use a fake user id here.

Reply:
Oh I didn't know it was a real one. I took the example from the spec

Reply:
Hmm, Erik is in the spec, well I guess it doesn't matter that much but let's just use `example`, or `alice` like we usually do.